api/db/changelog/db.changelog-157-add-user-access-tier.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="changelog-157-add-user-access-tier">
+
+    <createTable tableName="user_access_tier">
+      <column name="user_access_tier_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
+        <constraints nullable="false"
+          foreignKeyName="fk_user_access_tier_user"
+          references="user(user_id)"
+          deleteCascade="true"/>
+      </column>
+      <column name="access_tier_id" type="bigint">
+        <constraints nullable="false"
+          foreignKeyName="fk_user_access_tier_access_tier"
+          references="access_tier(access_tier_id)"
+          deleteCascade="true"/>
+      </column>
+      <column name="access_status" type="tinyint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="first_enabled" type="datetime">
+       <constraints nullable="false"/>
```

Comment:
This being non-nullable means we can only create rows that are initially `ENABLED`. Does that hold?

Reply:
Yes.  It does not make sense to create a new-DISABLED row.